app/scenes/Search/Search.js

```diff
@@ -229,9 +234,16 @@ class Search extends React.Component<Props> {
   };
 
   render() {
-    const { documents, notFound, location } = this.props;
+    const { documents, notFound, location, collections } = this.props;
     const results = documents.searchResults(this.query);
-    const showEmpty = !this.isFetching && this.query && results.length === 0;
+    const collectionResults = !this.collectionId
```

Comment:
As noted below – just call `const collectionResults = this.props.collections.search(this.query)` directly here 😄